src/lib/y2partitioner/widgets/format_and_mount.rb

```diff
@@ -612,6 +612,19 @@ def refresh
         self.value = @controller.encrypt
       end
 
+      # validates if it can be encrypted
+      def validate
+        return true unless value
+
+        # FIXME: have generic check when new partition is created if it is valid
+        # like e.g. if fs fits to partition, and others
+        # size number is from bsc1065071#c5
+        return true if @controller.blk_device.size > ::Y2Storage::DiskSize.MiB(2)
+
+        ::Yast::Popup.Error(_("For encryption partition have to be bigger then 2 MiB."))
```

Comment:
I little bit hard to read. What about "Only devices bigger than 2 MiB can be encrypted"